jenkins/ci.suse.de/cloud-ardana8.yaml

```diff
@@ -131,20 +131,42 @@
         - '{ardana_job}'
 
 - project:
-    name: cloud-ardana8-job-std-3cp-devel-staging-updates-x86_64
+    name: cloud-ardana8-job-entry-scale-kvm-x86_64
+    ardana_job: '{name}'
+    ardana_env: cloud-ardana-ci-slot
+    cloudsource: stagingcloud8
+    scenario_name: entry-scale-kvm
+    clm_model: standalone
+    controllers: '3'
+    sles_computes: '2'
+    rhel_computes: '0'
+    ses_enabled: true
+    ses_rgw_enabled: false
+    tempest_filter_list: >-
+      keystone,swift,glance,cinder,neutron,nova,barbican,lbaas,fwaas,vpnaas,
+      designate,heat,ceilometer,magnum,freezer,monasca
+    triggers: []
+    jobs:
+        - '{ardana_job}'
+
+- project:
+    name: cloud-ardana8-job-entry-scale-kvm-update-x86_64
     ardana_job: '{name}'
     ardana_env: cloud-ardana-ci-slot
     cloudsource: develcloud8
     update_after_deploy: true
     update_to_cloudsource: stagingcloud8
-    scenario_name: standard
+    scenario_name: entry-scale-kvm
     clm_model: standalone
     controllers: '3'
-    sles_computes: '1'
+    sles_computes: '2'
+    rhel_computes: '0'
     ses_enabled: true
     ses_rgw_enabled: false
-    triggers:
-     - timed: 'H H * * *'
+    tempest_filter_list: >-
```

Comment:
Something is not right here as the >- should be stripping off the trailing whitespace, and the leading whitespace should be being ignored... However the jobs shows a failure for running the designate filter (https://ci.suse.de/blue/organizations/jenkins/cloud-ardana9-job-entry-scale-kvm-x86_64/detail/cloud-ardana9-job-entry-scale-kvm-x86_64/175/pipeline/252#step-253-log-8) as follows:
[2019-05-01T07:11:05.537Z] Running: ansible-playbook run-tempest.yml -e @input.yml -e tempest_run_filter= designate
[2019-05-01T07:11:06.360Z] ERROR! the playbook: designate could not be found

That error suggests that the whitespace before the designate at the start of the above line got included in the value of the filters list... So the previous pattern using a quote string with \ continuation may have been correct... My bad...

Reply:
No worries, I reverted the change :)